switchapi/switch_pd_port.c

```diff
@@ -35,14 +34,16 @@ enum switch_tuntap_type {
 // Typedefs to compile stuff
 typedef uint16_t switch_pd_table_id_t;
 typedef uint16_t switch_pd_action_id_t;
-typedef switch_status_t bf_status_t;
+typedef switch_status_t tdi_status_t;
 typedef switch_status_t switch_pd_status_t;
 #define SWITCH_PD_STATUS_SUCCESS 0
 
 switch_status_t switch_pd_device_port_add(switch_device_t device,
     switch_dev_port_t dev_port,
     switch_uint32_t mtu)
 {
+//TODO: Possibly a dead code, confirm and remove
```

Comment:
It may well be dead code. PD is the Protocol Dependent interface, dating back to the earliest P4 implementations, which generated APIs based on your P4 program. I'm pretty sure it's been superseded.

Reply:
Will confirm with the Indian team before removing it, for now have added a TODO